plugin/edge/geo.go

```diff
@@ -0,0 +1,41 @@
+package edge
```

Comment:
good for now. I'm sure there is a lib we could use to do the same / be optimized